subworkflows/local/singular_demultiplex/main.nf

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env nextflow
+
+//
+// Demultiplex Singular Genomics bases data using sgdemux
+//
+
+include { SGDEMUX } from "../../../modules/nf-core/sgdemux/main"
+
+workflow SINGULAR_DEMULTIPLEX {
+    take:
+        // Input fastq's must be bgzipped for compatibility with sgdemux
+        // samplesheet.csv must be a two column csv = Sample_Barcode,Sample_ID)
+        ch_flowcell     // [[id:"", lane:""],samplesheet.csv, path/to/bases/files]
```

Comment:
```suggestion
        ch_flowcell     // [[id:"", lane:""],samplesheet.csv, path/to/fastq/files]
```

Reply:
Fixed in 125fff34b177f5af359fff6e5b6d5fd227bf1800